src/circl/hpke/vectors_test.go

```diff
@@ -15,8 +15,8 @@ import (
 
 func TestVectors(t *testing.T) {
 	// Test vectors from
-	// https://github.com/cfrg/draft-irtf-cfrg-hpke/blob/draft-irtf-cfrg-hpke-07/test-vectors.json
-	vectors := readFile(t, "testdata/vectors_v07.json")
+	// https://github.com/cfrg/draft-irtf-cfrg-hpke/blob/master/test-vectors.json
+	vectors := readFile(t, "testdata/vectors_v08_779d028.json")
```

Comment:
Did something go wrong with checking in updated files? vectors_v07.json was updated while the new name from https://github.com/cloudflare/circl/pull/222 is not present.

Reply:
Hmm... I'm not sure I understand the question here. I just applied a patch from CIRCL's change.

Reply:
Sorry I should have been clearer, this vectors_test.go file is fine, however the actual test vector itself has the wrong filename:
```
$ git log -p --stat -w -n1 origin/caw/ech-10
...
 src/circl/hpke/testdata/vectors_v07.json |  2 +-
```
Try `git mv src/circl/hpke/testdata/vectors_v07.json src/circl/hpke/testdata/vectors_v08_779d028.json`

Reply:
🤦 oops. Thanks for catching that!